drivers/storage/partmgr/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
`add_subdirectory(partmgr)` should be added to upper level `CMakeLists.txt` file, right? :sweat_smile: